benchmarks/htap/stats_schema.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE oltp_stats(
+    id UUID
+  , ts TIMESTAMP NOT NULL
```

Comment:
Just asking: do we have to use TIMESTAMP or an INT for an elapsed time since the start of the test would do it?

Reply:
This is in the stats database. The timestamp records when the stats entry was stored.